containers/README.md

```diff
@@ -0,0 +1,123 @@
+<!--
+    Copyright 2023 TII (SSRC) and the Ghaf contributors
+    SPDX-License-Identifier: CC-BY-SA-4.0
+-->
+
+# Containers
+
+Follow the instructions below to run various services inside Docker containers.
+
+## Hydra Container
+
+### Run Hydra in Container
+
+Custom configuration can be set with the `confs/hydra.local`.
+See the default configuration in the [confs/hydra.default](confs/hydra.default).
+
+[More detailed instructions to configure and run the Hydra instance](README.hydra)
```

Comment:
For more detailed information on configuring and running the Hydra instance, see [README.jagent](README.hydra).